server/models/liked_entity.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Liked_Entity = sequelize.define('Liked_Entity', {
+    user_id: DataTypes.INTEGER,
+    entity_id: DataTypes.INTEGER
+  }, {});
+  Liked_Entity.associate = function(models) {
+    // associations can be defined here
```

Comment:
Same question here from above haha, is this a join table since it holds both user_id & entity_id

Reply:
This table is supposed to be representing 'Likes'. It's a M2M relationship, so it's pretty much a join table.